karma.conf.js

```diff
@@ -0,0 +1,100 @@
+// Karma configuration
+// Generated on Fri Jul 29 2016 14:01:07 GMT+1000 (AEST)
+var webpackConfig = require('./webpack.config.js');
+
+module.exports = function(config) {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'spec/helpers/**/*.js',
+      'http://cdn.jsdelivr.net/g/lodash@2.4.2(lodash.underscore.min.js),handlebarsjs@1.3.0,jquery@2.2.4,momentjs@2.9.0,bootstrap@2.3.2',
+      'http://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js',
+      'spec/**/*_spec.js'
+    ],
+
+    // list of files to exclude
+    exclude: [
+    ],
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      'spec/helpers/**/*.js': ['webpack'],
+      'spec/**/*_spec.js': ['webpack']
+    },
+
+    webpack: {
+      // karma watches the test entry points
+      // (you don't need to specify the entry option)
+      // webpack watches dependencies
+
+      // webpack configuration
+      module: webpackConfig.module,
+      resolveLoader: webpackConfig.resolveLoader,
+      resolve: webpackConfig.resolve,
+      externals: webpackConfig.externals
+    },
+
+    webpackServer: {
+      noInfo: true // Suppress all webpack messages, except errors
+    },
+
+    webpackMiddleware: {
+      // webpack-dev-middleware configuration
+      // i. e.
+      stats: 'errors-only'
+    },
+
+    plugins: [
+      require("karma-webpack"),
+      require("karma-jasmine"),
+      require("karma-phantomjs-launcher"),
+      require("karma-chrome-launcher")
+    ],
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['dots'],
+
+    // web server port
+    port: 9876,
+
+    // enable / disable colors in the output (reporters and logs)
+    colors: true,
+
+    // level of logging
+    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+    logLevel: config.LOG_INFO,
+
+    // enable / disable watching file and executing tests whenever any file changes
+    autoWatch: true,
```

Comment:
If you weren't a proponent for TDD, and didn't want an additional server/process running?